woocommerce/woocommerce-android
# [Better Tablet Products] Select the first product by default in tablet

Closes: #10780
Closes: #10788


### Description

The PR:
* Adds selection fo the first loaded product on the two-pane layout
* Adds highlighting of a selected product on the two-pane layout 

### Testing instructions

* Make sure that with flag OFF, both tablet and phone work in single pane mode
* Make sure that with the flag ON, the tablet works in two-pane mode, and the first loaded product is selected. Notice also highlighted background

Also, pay attention that partially the same code is used:
* Products
* Add product
* Group Product
* Add product to group
* Add product
So no changes should happen there

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/7efc1804-74a8-47e8-99e1-a549db00a3f6
